docs/content/preview/admin/ysql-dump.md

```diff
@@ -161,13 +171,13 @@ This option disables the use of dollar quoting for function bodies, and forces t
 
 #### --disable-triggers
 
```

Comment:
Format line 166 : Dump data as `INSERT` statements with explicit column names (`INSERT INTO table (column, ...) VALUES ...`). This makes restoration very slow; it is mainly helpful for making dumps that can be loaded into non-YugabyteDB databases. However, as this option generates a separate statement for each row, an error in reloading a row causes only that row to be lost rather than the entire table contents.